packages/graphql-tool-utilities/package.json

```diff
@@ -20,6 +20,9 @@
     "url": "https://github.com/Shopify/quilt/issues"
   },
   "homepage": "https://github.com/Shopify/quilt/blob/main/packages/graphql-tool-utilities/README.md",
+  "engines": {
+    "node": ">=12.14.0"
+  },
   "dependencies": {
     "apollo-codegen-core": "0.36.5",
     "core-js": "^3.0.0",
```

Comment:
Remove?

Reply:
This keeps getting added always

Reply:
Yep. I removed the usage, but somehow missed the dependency.